test-clients/src/main/java/com/hedera/services/bdd/suites/perf/MixedOpsLoadTest.java

```diff
@@ -0,0 +1,287 @@
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_SIGNATURE;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TOKEN_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TOPIC_ID;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PAYER_ACCOUNT_NOT_FOUND;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PLATFORM_TRANSACTION_NOT_CREATED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKENS_PER_ACCOUNT_LIMIT_EXCEEDED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOKEN_NOT_ASSOCIATED_TO_ACCOUNT;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TOPIC_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.TRANSACTION_EXPIRED;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.UNKNOWN;
+
+public class MixedOpsLoadTest extends LoadTest {
+	private static final Logger log = LogManager.getLogger(MixedOpsLoadTest.class);
+	private static final int NUM_SUBMISSIONS = 100;
+	private final ResponseCodeEnum[] permissiblePrechecks = new ResponseCodeEnum[] {
+			BUSY, DUPLICATE_TRANSACTION, PLATFORM_TRANSACTION_NOT_CREATED, UNKNOWN
+	};
+
+	private static String sender = "sender";
+	private static String receiver = "receiver";
+	private static String submitKey = "submitKey";
+	private static String topic = "topic";
+	private static String token = "token";
+	private static String schedule = "schedule";
+	private static int messageSize = 1024;
+
+	public static void main(String... args) {
+		parseArgs(args);
+		MixedOpsLoadTest suite = new MixedOpsLoadTest();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runMixedOps()
+		);
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+	protected HapiApiSpec runMixedOps() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		Random r = new Random();
+		AtomicInteger tokenId = new AtomicInteger(0);
+		AtomicInteger scheduleId = new AtomicInteger(0);
+
+		Supplier<HapiSpecOperation[]> mixedOpsBurst = () -> new HapiSpecOperation[] {
+				cryptoTransfer(
+						tinyBarsFromTo(sender, receiver, 1L))
+						.noLogging()
+						.payingWith(sender)
+						.signedBy(GENESIS)
+						.suppressStats(true)
+						.fee(ONE_HBAR)
+						.hasKnownStatusFrom(SUCCESS,
+								OK,
+								INSUFFICIENT_PAYER_BALANCE,
+								UNKNOWN,
+								TRANSACTION_EXPIRED)
+						.hasRetryPrecheckFrom(BUSY,
+								DUPLICATE_TRANSACTION,
+								PLATFORM_TRANSACTION_NOT_CREATED,
+								INVALID_SIGNATURE,
```

Comment:
- having `UNKNOWN` in known status list is counter-intuitive
- `INVALID_SIGNATURE` in retry precheck doesn’t seem right

Reply:
@qnswirlds Moved `INVALID_SIGNATURE` to precheck instead of retry precheck.  
As discussed, I believe `UNKNOWN` is added to all Load test to prevent the test stopping in the middle with this status. Thanks

Reply:
@qnswirlds  removed `INVALID_SIGNATURE` and validated test passes. Thanks https://hedera-hashgraph.slack.com/archives/C018CBQBTGB/p1615568216354800